docs/user-guide/plugins.md

```diff
@@ -7,6 +7,7 @@ Plugins are rules built by the community that support methodologies, toolsets, *
 ### Selector
 
 - [`stylelint-selector-bem-pattern`](https://github.com/davidtheclark/stylelint-selector-bem-pattern): Specify a BEM pattern for selectors (incorporates [postcss-bem-linter](https://github.com/postcss/postcss-bem-linter)).
+- [`stylelint-selector-max-depth`](https://github.com/dryoma/stylelint-selector-max-depth): Limit selectors depth. In terms of this rule, selector depth is how many levels of HTML structure (not necesserily direct descendants) the selector is reflecting. For example, `.foo h3 + a > span` has the depth of 3: `.foo` is level 1, `a` (and `h3`, they are siblings) is level 2, and `span` is level 3.
```

Comment:
That looks good to me. Could add an apostrophe, though: `Limit selectors' depth.` @jeddy3 What do you think?